Extension/src/LanguageServer/ui.ts

```diff
@@ -426,14 +435,15 @@ export class LanguageStatusUI {
     private showConfigureIntelliSenseButton: boolean = false;
     private configureIntelliSenseTimeout?: NodeJS.Timeout;
 
-    public async ShowConfigureIntelliSenseButton(show: boolean, client?: Client): Promise<void> {
+    public async ShowConfigureIntelliSenseButton(show: boolean, client: Client, configurationType?: ConfigurationType, sender?: string): Promise<void> {
+        if (configurationType !== undefined && sender !== undefined && this.showConfigureIntelliSenseButton && !show) { // Only log telemetry when the button is being hidden.
```

Comment:
I think it would be good to log when the button is shown because we may not see them ever fix it.  Can you add the "show" property to the Properties for this event?

With this in mind, there is another call missed in `client.ts` around line 2775 that should pass in the type and sender (it will actually be a few lines down since lines were added in this PR)